package.json

```diff
@@ -16,17 +16,14 @@
       "jquery": true
     },
     "globals": {
-      // 1. Table names
```

Comment:
Yeah, I was just curious what errors the comments were causing. Thanks for clarification!